File: Java_Practice/Python/dma_program3.py
#Input n numbers dynamically and print sum
n = int(input("Enter number of elements: "))
arr = []

sum_val = 0
for i in range(n):
    val = int(input(f"Enter element {i+1}: "))
    arr.append(val)
    sum_val += val

print("Sum =", sum_val)

File: Java_Practice/CPP/dma_program3.cpp
//Input n numbers dynamically and print sum
#include <iostream>
using namespace std;

int main() {
    int n, sum = 0;
    cout << "Enter number of elements: ";
    cin >> n;

    int *arr = new int[n];

    for(int i=0;i<n;i++){
        cout << "Enter element " << i+1 << ": ";
        cin >> arr[i];
        sum += arr[i];
    }

    cout << "Sum = " << sum << endl;
    delete[] arr;
    return 0;
}
